locales/en.yml

```diff
@@ -70,3 +70,22 @@ en:
       description: Discover the story of our Logo
       page_title: Logo
     title: Logo of Ruby Nord
+  talks:
+    description: |
+      Find all the past talks from our Apéros Ruby.
+      Whether you're a beginner or a confirmed developer, feel free to give a talk and share your experience with the community.
+    level:
+      all: all levels
```

Comment:
I wouldn't use `all` to be honest. I would just use `beginner` because it works whatever your background is.
I really think the difference between `all` and `beginner` depends only of your background.

Example: I'm working with Ruby, mostly in a web environment. I never used `RubyMotion` in this web environment. An introduction to `RubyMotion` would be `all` for me whereas it would be `beginner` for someone who is used to do mobile development in ruby.

If you introduce a gem/subject and anyone can understand it, I think it should be `beginner` (even if confirmed ruby users might not know the gem/subject). 

I really think the level should be there just to specify the minimum skills you need to have to understand a talk.

`soft`: no ruby knowledge
`beginner`: small ruby knowledge 
`confirmed`: a bit more ruby knowledge

What do you think? Some others have feelings about this maybe?